src/monero_key.c

```diff
@@ -253,7 +253,8 @@ int monero_apdu_put_key() {
     unsigned char pub[32];
     unsigned char sec[32];
 
-    if (G_monero_vstate.io_length != (32 * 2 + 32 * 2 + 95)) {
+    // priv/pub view key + priv/pub spend key + address + option
+    if (G_monero_vstate.io_length != (32 * 2 + 32 * 2 + 95 + 1)) {
```

Comment:
Should be better if written as :
```
    //option +  priv/pub view key + priv/pub spend key + base58 public address
    if (G_monero_vstate.io_length != (1+32 * 2 + 32 * 2 + 95)) 
```
to follow the real stream order

Reply:
Done in commit 499f41fc0341048d34947fc41c2cffd25043ebd3.